### ISSUE TITLE
clarify details

### ISSUE THREAD

**Issue by ruida** [wangjunchi/LLMSeg#7]
1)  /prepare_datasets/prepare_egoobjects.py

# test set not available for ReasonSeg
available_dataset_type = ["train", "validation", "test"]

def get_all_samples() -> List[str]:
    
    json_path = "/home/leikel/junchi/ReasonCOCO/post_processing/split"

Could you leave more details of ReasonCOCO/post_processing/split?       Where I can get the train, validation, test json files?  
Do you use EgoObjectsV1J_images.zip as the source image files? 

2)  Prepare the pretrained models
Please follow the [instruction](https://github.com/haotian-liu/LLaVA/blob/main/docs/MODEL_ZOO.md) to merge the LLaVA delta weights. We use LLaVA-lightning-7B-v1 checkpoint.

Could you leave me more detail on how to merge weights?   The link is not clear.     

Thanks.
